cloud/wfm-rest-api/src/impl/ApiController.ts

```diff
@@ -95,6 +95,12 @@ export class ApiController<T> {
       return Bluebird.reject(error);
     }
 
+    req.body.id = req.body.id || req.params.id;
```

Comment:
I'm not sure we should be flexible here. Which way do we prefer? 
Documentation states that api for put is `PUT /`. If we want to expose id as param let's update this.